includes/class-wc-gateway-cardknox.php

```diff
@@ -114,6 +114,11 @@ public function __construct()
         $this->enable_3ds              = $this->get_option('enable-3ds');
         $this->threeds_env             = $this->get_option('3ds-env');
 
+        $this->applicable_countries   = $this->get_option('applicable_countries'); // Add this Code
```

Comment:
What is comment for?

Reply:
Hello @mstein-ck 

We have built the functionality of a country-wise credit card display. This functionality is also already implemented on Google Pay and Apple Pay but not the credit cards. So, we have a build a common field structure and work this functionality.

Please let me know if anything else

Thank you.